experimental/ModelBuilder/test/BenchMatMulVectorGPU.cpp

```diff
@@ -85,6 +90,86 @@ static void addLoweringPasses(mlir::PassManager &pm,
   pm.addPass(mlir::createConvertVulkanLaunchFuncToVulkanCallsPass());
 }
 
+/// Allocation callback for allocation workgroup local memory.
+static Value allocateWorkgroupMemory(OpBuilder &b, SubViewOp subview,
```

Comment:
This seems to be similar to what is being done in IREE codegen. Can we just expose that directly?